### ISSUE TITLE
[UI Fix]: Home page is not responsive

### ISSUE THREAD

**Issue by Rahul7raj** [Bitbox-Connect/Bitbox#29]
## Issue: Home Page is Not Responsive
The home page does not adapt well to different screen sizes and devices, leading to elements overlapping or being misaligned when viewed on mobile or tablet devices.

## Recording
https://github.com/user-attachments/assets/b976b512-b1ef-4a5b-9e0e-182ddde77fbc

## Proposed Solution
I can work on making the home page fully responsive by adjusting the CSS styles and layout structure to ensure it looks and functions correctly across various screen sizes.

**Comment by Anuj3553** [comment]
Okay @Rahul7raj, I'll assigning this issue to you.

**Comment by Rahul7raj** [comment]
@Anuj3553 can you please guide me on how can I run the application? it's not mentioned in readme md

**Comment by Anuj3553** [comment]
> @Anuj3553 can you please guide me on how can I run the application? it's not mentioned in readme md

Okk @Rahul7raj,

Step1 -> Open first terminal and type -> cd client, npm i and then npm run dev
Step1 -> Open second terminal and type -> cd server, npm i and then nodemon index.js

**Comment by Rahul7raj** [comment]
Hey @Anuj3553 the pages are not getting rendered properly. I attached the video for the reference, 

https://github.com/user-attachments/assets/54495c0f-9159-4746-829f-bfa77cd87b70

Is this only I am facing ? or is it real issue ?

**Comment by Anuj3553** [comment]
This issue is created by contributors. Now it's made a issue.

**Comment by Rahul7raj** [comment]
So it is better to fix that issue first, I think the issue #33 was raised for that. I will raise the PR for this after #33 issue was closed.

**Comment by Anuj3553** [comment]
Hyy @Rahul7raj, I can edit this issue title so you can work for this issue or you can create a new issue. Let me tell you what you want to do.

**Comment by Rahul7raj** [comment]
Regarding the rendering issue, I think @Bhum-ika is already working on it since she raised issue #33 . So let's wait for her PR to be merged.

**Comment by Anuj3553** [comment]
> Regarding the rendering issue, I think @Bhum-ika is already working on it since she raised issue #33 . So let's wait for her PR to be merged.

Yes, she texted me in discord that she gonna finished her work soon.
